static/style.css

```diff
@@ -13,6 +13,27 @@ body {
   color: #333;
 }
 
+nav {
+  position: fixed;
+  right: 0.5rem;
+  top: 5%;
+  max-height: 90%;
+  overflow: scroll;
+
+  border-left: 1px solid firebrick;
+  padding-left: 0.5rem;
+  padding-right: 1rem;
+  
+}
+nav:before {
+  content: "Contents";
+  font-weight: bold;
+}
+nav ol {
+  list-style-type:none;
+  padding-left: 1rem;
```

Comment:
@dan-zheng this is where you would change indentation width.

My skills go:

- CSS: I am merely bad at, but I know how many things work
- HTML: Making it up as I go along
- Javascript: I literally don't know what semicolons do.

Reply:
Awesome qualifications, thank you for interviewing. When could you start at our $150k full-stack development position?

Reply:
Great, thank you!